.mergify.yml

```diff
@@ -22,4 +27,11 @@ pull_request_rules:
     actions:
         comment:
           message: This pull request is now in conflicts. Could you fix it @{{author}}? 🙏
+        label:
+          add:
+            - s:revision-needed
+          remove:
+            - s:accepted
+            - s:in-progress
+            - s:review-needed
```

Comment:
Interesting! Does this ensure it's detects only the last commit of a PR that fails? Otherwise I could imagine a previous workflow fails although you fixed it already with a new commit and it always jumps to "revision needed".

Reply:
AFAIK mergify always considers the latest workflow runs, but I guess we'll have to see in practice if this holds true :)